src/NLog.Extensions.Logging/NLogLogger.cs

```diff
@@ -114,11 +137,7 @@ private void CaptureEventId(EventId eventId, LogEventInfo eventInfo)
                 if (!ReferenceEquals(eventIdPropertyNames.Item1, eventIdSeparator))
                 {
                     // Perform atomic cache update of the string-allocations matching the current separator
-                    eventIdPropertyNames = new Tuple<string, string, string>(
-                        eventIdSeparator,
-                        string.Concat("EventId", eventIdSeparator, "Id"),
-                        string.Concat("EventId", eventIdSeparator, "Name"));
-                    _eventIdPropertyNames = eventIdPropertyNames;
+                    _eventIdPropertyNames = CreateEventIdPropertyNames(eventIdSeparator);
```

Comment:
@snakefoot did I broke the atomic update? See https://travis-ci.org/NLog/NLog.Extensions.Logging/builds/310224901?utm_source=github_status&utm_medium=notification

Reply:
ow found it :)